Automate/Speckle.Automate.Sdk/AutomationContext.cs

```diff
@@ -137,7 +158,7 @@ public void SetContextView(List<string>? resourceIds = null, bool includeSourceM
 
     if (linkResources.Count == 0)
     {
-      throw new Exception("We do not have enough resource ids to compose a context view");
+      throw new SpeckleException("We do not have enough resource ids to compose a context view");
```

Comment:
Maybe a more specific exception would be desirable here?

Reply:
how about ?

```c#
public class SpeckleContextViewException : SpeckleException { }
```